releasenotes/notes/0.13/defaults_TranspileConfig-4109a90c278d46df.yaml

```diff
@@ -0,0 +1,30 @@
+---
+upgrade:
+  - |
+    The ``TranspileConfig`` class which was previously used to set
```

Comment:
Should TranspileConfig have been deprecated, or was it considered not user-facing?

Reply:
I don't think it was really user facing. I think it was only ever really used by `transpile` and `transpile_circuits` when instantiating the preset pass managers, but I could be wrong. I just rewrote this based on: https://github.com/Qiskit/qiskit-terra/pull/4057/files#diff-98dbaa07a448764c4971f422304273ac